deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -101,3 +108,93 @@ def make_vuln_callback(pattern):
     regex = re.compile(r'{}{}'.format(vulnerability_detector_prefix, pattern))
 
     return lambda line: regex.match(line) is not None
+
+
+def load_db(db_path):
+    conn = sqlite3.connect(db_path)
+    cursor = conn.cursor()
+    return conn, cursor
+
+
+def make_query(db_path, query_list):
+    """
+    Makes a query to the database in db_path for each item in query_list
+    """
+    try:
+        db = load_db(db_path)
+        for item in query_list:
+            db[1].execute(item)
+        db[0].commit()
+    except sqlite3.Error as error:
+        print("Failed perform sqlite query\n", error)
+        raise error
+    finally:
+        db[1].close()
+        db[0].close()
```

Comment:
Watch out wit this. One error is transformed in several errors. If you are not going to handle the exception, just do this: 
```suggestion
    try:
        db = load_db(db_path)
        for item in query_list:
            db[1].execute(item)
        db[0].commit()
    finally:
        db[1].close()
        db[0].close()
```

Reply:
Done in ff83ea8